src/libcore/num/f32.rs

```diff
@@ -128,10 +128,18 @@ pub mod consts {
     #[stable(feature = "rust1", since = "1.0.0")]
     pub const LOG2_E: f32 = 1.44269504088896340735992468100189214_f32;
 
+    /// log<sub>2</sub>(10)
+    #[unstable(feature = "extra_log_consts", issue = "50539")]
```

Comment:
Please use the tracking issue number and not the PR number.

Reply:
My mistake; that's fixed now.